examples/Building-and-deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb

```diff
@@ -115,43 +115,15 @@
    "source": [
```

Comment:
<p>Maybe we could add the section <code>Exporting query (user) model</code> here to keep the logical flow. We can add a sentence like <code>We export the query tower to use it later during the deployment&nbsp;stage.</code>  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>